lume_model/models/torch_module.py

```diff
@@ -243,3 +243,143 @@ def register_to_mlflow(
             save_jit,
             **kwargs,
         )
+
+
+class PriorModel(torch.nn.Module):
```

Comment:
Lets rename this to FixedVariableModel or something similar as this feature is useful outside the context of Xopt

Reply:
should also remove Xopt references in the docstring